docs/guides/kuma.md

```diff
@@ -0,0 +1,233 @@
+```
+
+Now the knative-serving pods are running in the mesh and ready to work with mtls Strict.
+
+## Configure Kuma mTLS to Strict mode
+
+We are going to modify the Kuma Mesh to enable the mTLS with Strict mode:
+
+```console
+kubectl apply -f - <<EOF                            
+apiVersion: kuma.io/v1alpha1
+kind: Mesh
+metadata:
+  name: default
+spec:
+  mtls:
+    enabledBackend: ca-1
+    backends:
+      - name: ca-1
+        type: builtin
+        mode: STRICT
+EOF
+```
+
+### Test Service
+
+Deploy a Knative Service:
+
+!!! Info "Knative Services"
+    The knative services also needs [port-exclusions](https://kuma.io/docs/dev/reference/kubernetes-annotations/#traffic-kuma-io-exclude-inbound-ports)
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: helloworld-go
+  namespace: default
+spec:
+  template:
+    metadata:
+      annotations:
+        traffic.kuma.io/exclude-inbound-ports: "8012"
+    spec:
+      containers:
+        - image: gcr.io/knative-samples/helloworld-go
+          env:
+            - name: TARGET
+              value: Go Sample v1
+EOF
+```
+
+Wait for the service to be ready, then get it's exposed URL. The URL should be a sub-domain of the configured CNAME:
+
+```console
+kubectl get ksvc
+NAME URL LATESTCREATED LATESTREADY READY REASON
+helloworld-go   http://helloworld-go.default.10.101.62.158.nip.io   helloworld-go-00001   helloworld-go-00001   True 
+
+```
+
+A call to the URL using a web browser or `curl` should return a successful text response:
+
+```console
+curl -v http://helloworld-go.default.10.101.62.158.nip.io
+*   Trying 10.101.62.158:80...
+* Connected to helloworld-go.default.10.101.62.158.nip.io (10.101.62.158) port 80 (#0)
+> GET / HTTP/1.1
+> Host: helloworld-go.default.10.101.62.158.nip.io
+> User-Agent: curl/7.81.0
+> Accept: */*
+> 
+* Mark bundle as not supporting multiuse
+< HTTP/1.1 200 OK
+< Content-Type: text/plain; charset=utf-8
+< Content-Length: 20
+< Connection: keep-alive
+< Date: Wed, 30 Mar 2022 11:22:29 GMT
+< X-Kong-Upstream-Latency: 1
+< X-Kong-Proxy-Latency: 1
+< Via: kong/2.7.1
+< 
+Hello Go Sample v1!
+* Connection #0 to host helloworld-go.default.10.101.62.158.nip.io left intact
+```
+
+## Kuma UI
+
+Kuma includes a dashboard that show some util information, to see it we are going to do a port-forward:
+
+```console
+kubectl port-forward svc/kuma-control-plane -n kuma-system 5681:5681
+```
+
+Now in a web browser we can open <http://localhost:5681/gui> and see the dashboard:
+
+Kuma Global Overview
+![Kuma Global Overview](../../assets/images/kuma/kuma-global-overview.png)
+
+Kuma Mesh Overview
+![Kuma Mesh Overview](../../assets/images/kuma/kuma-mesh-overview.png)
+
+Kuma Gateway
+![Kuma Gateway](../../assets/images/kuma/kuma-gateway.png)
+
+Kuma Data Plane Proxies
+![Kuma Data Plane Proxies](../../assets/images/kuma/kuma-data-plane-proxies.png)
```

Comment:
Fixing the asset location (mkdocs consola was complaining).
Also, I feel like removing most of the UI images, maybe leaving one.

The reason is that they don't add much information and will get eventually stale as Kuma releases new versions. However they look good, hence leaving one.


```suggestion
To access the UI opening <http://localhost:5681/gui> with a web browser:

![Kuma Global Overview](../assets/images/kuma/kuma-global-overview.png)

```